pages/app/index.tsx

```diff
@@ -80,6 +71,10 @@ function App() {
   );
 }
 
+// The VR class needs to be set before any React rendering takes occurs.
+const vrDisabled = window.location.hash.includes('visualRefresh=false');
+document.getElementById('app')?.classList.toggle('awsui-visual-refresh', !vrDisabled);
```

Comment:
I also see the benefit of not using the optional chaining here.

Reply:
Yup, will update this.